datastream-kafka/src/main/java/com/linkedin/datastream/kafka/KafkaProducerWrapper.java

```diff
@@ -59,6 +63,9 @@
 
   private static final int TIME_OUT = 2000;
   private static final int MAX_SEND_ATTEMPTS = 10;
+  private static final int SEND_TIME_OUT = 5000;
```

Comment:
We need to come up with reasonable numbers here. Maybe test a bit more to understand how the system behaves if we change these.